Content.Shared/Shipyard/SharedShipyardSystem.cs

```diff
@@ -7,21 +7,33 @@
 
 namespace Content.Shared.Shipyard;
 
+// Note: when adding a new ui key, don't forget to modify the dictionary in SharedShipyardSystem
 [NetSerializable, Serializable]
 public enum ShipyardConsoleUiKey : byte
 {
     Shipyard,
     Security,
     BlackMarket,
     Expedition,
-    Scrap
+    Scrap,
+    // Not currently implemented. Could be used in the future to give other factions a variety of shuttle options, like nukies, syndicate, or for evac purchases.
```

Comment:
i think we can change my comment here since this actually is implemented now hahah! When i originally wrote this comment was before I had even implemented security's shipyard